blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DBasicMetricsRW.java

```diff
@@ -102,6 +102,9 @@ public void insertMetrics( Collection<IMetric> metrics ) throws IOException {
                 }
 
                 futures.put( locator, rawIO.insertAsync( metric ) );
+
+                // this is marking  metrics_strings & metrics_full together.
+                Instrumentation.markFullResMetricWritten();
```

Comment:
If you move this up here, is it possible we will be double-counting metrics in error cases? This means, the fullResMetricWritten count possible includes the erroneous ones?